src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -903,9 +1205,9 @@
         internal enum RefKindCompareMode
         {
             /// <summary>
-            /// Ref parameter modifiers are ignored.
+            /// All ref modifiers are considered equivalent.
             /// </summary>
-            DoNotConsiderDifferences = 0,
+            RefMatchesOutInRefReadonly = 0,
```

Comment:
>RefMatchesOutInRefReadonly [](http://example.com/codeflow?start=12&length=26)

It feels like "RefOutInRefReadonlyMatch" is going to reflect the meaning more accurately.